Documents/API.md

```diff
@@ -132,30 +158,50 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
     * `POST` - Upload HMDA data to submission
 
 
-* `/institutions/<institution>/summary`
-    * `GET` - Summary for Financial Institution, including filing information
+* `/institutions/<institution>/filings/<period>/submissions/<submissionId>/edits`
+    * `GET`  - List of all edits for a given submission, grouped by edit type
 
     Example response, with HTTP code 200:
 
     ```json
     {
-      "id": "12345",
-      "name": "First Bank",
-      "filings": [
-        {
-          "period": "2017",
-          "institutionId": "12345",
-          "status": "not-started"
-        },
-        {
-          "period": "2016",
-          "institutionId": "12345",
-          "status": "completed"
-        }
-      ]
+      "syntactical": {
```

Comment:
In the [front-end documentation](https://github.com/cfpb/hmda-platform-ui/blob/c7590b1e6bd3243a8cd7909463ab8e2dc8c046a5/server/json/quality.json), quality edits have a "verification" key for each LAR. Is that coming in a future PR? (If so, let's make sure there's an issue documenting that it needs to be done)